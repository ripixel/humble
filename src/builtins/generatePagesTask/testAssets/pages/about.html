<html>
  <body>
    {{> header}}
    <h1>About</h1>
    <p>This is the about page.</p>
    {{> footer}}
  </body>
</html>
