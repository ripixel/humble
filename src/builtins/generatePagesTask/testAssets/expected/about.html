<html>
  <body>
    <header><h1>Header Partial</h1></header>
    <h1>About</h1>
    <p>This is the about page.</p>
    <footer><p>Footer Partial</p></footer>
  </body>
</html>
